shared/vendor/js/JitsiMeetExternalAPI.js

```diff
@@ -0,0 +1,411 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
```

Comment:
I don't think we can put our license here, since this is adapted from jitsi-meet project?